pkg/cli/install.go

```diff
@@ -40,6 +46,14 @@ func (i *Install) Run(cmd *cobra.Command, args []string) error {
 		image = i.Image
 	}
 
+	i.DefaultServiceMemory = DEFAULT_SERVICE_MEMORY
+	if i.DefaultServiceMemoryString != "" {
+		quantity, err := resource.ParseQuantity(i.DefaultServiceMemoryString)
+		if err != nil {
+			return err
+		}
+		i.DefaultServiceMemory = quantity.Value()
+	}
```

Comment:
Trying to brainstorm through ways to improve this to not be as hacky. We like the UX of `acorn install -m 512M` vs `acorn install -m 536,870,912` so a string feels natural but it causes a couple of issues (namely two flags being present).

Reply:
This is because we want the API to be an int, but the flag to be a string, right?

Reply:
Yes, exactly. Its cumbersome to type out all 9 digits for a megabyte and go doesn't support the suffixes [natively like Cue does](https://cuelang.org/docs/tutorials/tour/types/numbers/).

Reply:
I think we reached a good middle ground here with the latest commits.